.pylintrc

```diff
@@ -144,7 +144,11 @@ disable=print-statement,
         R0903,
         R1705,
         R0123,
-        E0401
+        E0401,
+        E0110,
+        E1310,
+        C0123,
+        R0801
```

Comment:
I could see more suppression added what is the rationale ?

Reply:
2 more lint errors related to sanity test(re used few lines from functional test) are suppressed

Reply:
can you brief on the rationale ?